airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -3008,3 +3011,254 @@ def _format_schema_for_description(schema: Dict) -> List:
         )
         description.append(field_description)
     return description
+
+
+class BigQueryAsyncHook(GoogleBaseAsyncHook):
+    """Big query async hook inherits from GoogleBaseAsyncHook class and connects to the google Big query"""
```

Comment:
google Big query -> Google Big Query

also the parent is in the class so no need to mention that it inherits from GoogleBaseAsyncHook

Reply:
Done